content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+## Relevant works
+
+
+### 20th social science {#sec:intro-review-social}
+
+Social science has concerned itself with the stability, temporal evolution, and spatial variations and regularities of cultures since the start of its discipline.
+Émile Durkheim already, in his seminal study [-@durkheim_suicide:_2012], was looking at the regularities of suicide rates over the years, and the correlation of those rates with a partition of society into religion-related groups.
+The continuity of suicide over time, and the links between suicide rate and social group, he argued, suggest we should study suicide as a *social fact*:
```

Comment:
Ok, alors il y a un flou sur ce que j'introduis. Pour moi, Durkheim est un exemple de dualisme culture-cognition, que j'oppose à Maus & Giddens qui, au-delà d'une acceptation de l'importance du niveau culturel, montrent comment les deux s'imbriquent (plutôt que de vouloir les isoler comme étant autonomes). (Comme j'ai lu quelque part de la part d'un anthropologue, "the feat of social science has been to maintain the illusion of culture as separated from biology".) Donc ça s'attache à la question des liens cognition-culture.

Mais c'est effectivement une autre question que celle de la régularité/évolution temporelles et spatiales qui occupe Sperber. C'est tout mon défi : la première question est au fond celle qui m'intéresse, la deuxième est celle sur laquelle j'ai travaillé concrètement, et je voudrais convaincre le lecteur que la deuxième éclaire un peu la première.

Comme tu dis [plus bas](https://github.com/wehlutyk/thesis/pull/3#discussion_r124298484), l'intro des Kulturwissenschaften est pas claire, mais je voudrais pas non plus faire un exposé socio-anthropo plus long que ce que j'ai là, d'une part parce que je connais peu la littérature proprement Kulturwissenschaften, d'autre part parce que je crois que c'est pas trop le sujet. Oder?

Peut-être que ça serait mieux de commencer directement par Sperber & l'évolution culturelle, et passer l'introduction de Durkheim-Mauss-Bourdieu-Giddens dans une section fusionnée avec le 1.1.5 "Criticisms" (à renommer) ?